doc/sphinx-guides/source/installation/config.rst

```diff
@@ -19,14 +19,16 @@ Changing the Superuser Password
 
 The default password for the "dataverseAdmin" superuser account is "admin", as mentioned in the :doc:`installation-main` section, and you should change it, of course.
 
+.. _blocking-api-endpoints:
+
 Blocking API Endpoints
 ++++++++++++++++++++++
 
 The :doc:`/api/native-api` contains a useful but potentially dangerous API endpoint called "admin" that allows you to change system settings, make ordinary users into superusers, and more. The ``builtin-users`` endpoint lets people create a local/builtin user account if they know the ``BuiltinUsers.KEY`` value described below.
 
-By default, all APIs can be operated on remotely and a number of endpoints do not require authentication. https://github.com/IQSS/dataverse/issues/1886 was opened to explore changing these defaults, but until then it is very important to block both the "admin" endpoint (and at least consider blocking ``builtin-users``). For details please see also the section on ``:BlockedApiPolicy`` below.
+By default, all APIs can be operated on remotely and a number of endpoints do not require authentication. https://github.com/IQSS/dataverse/issues/1886 was opened to explore changing these defaults, but until then it is very important to block both the "admin" endpoint (and at least consider blocking ``builtin-users``). For details please see also the section on :ref:`:BlockedApiPolicy` below.
 
-It's also possible to prevent file uploads via API by adjusting the ``:UploadMethods`` database setting.
+It's also possible to prevent file uploads via API by adjusting the :ref:`:UploadMethods` database setting.
 
```

Comment:
It is very easy to lock yourself out if you don't use the commands in the correct order.
Could you add the full example here, so this is not left as an excersice to the admin?
You can copycat from https://github.com/IQSS/dataverse-kubernetes/blob/6f83afbeeb249542e87b9d04ae504d2ff319abfe/docker/dataverse-k8s/bin/bootstrap-job.sh#L60-L62 :wink:

Reply:
@poikilotherm good catch. Fixed in bc08e28